config/samples/storage_v1_csm_powerflex.yaml

```diff
@@ -55,7 +55,7 @@ spec:
       - name: snapshotter
         image: registry.k8s.io/sig-storage/csi-snapshotter:v8.1.0
       - name: csi-metadata-retriever
-        image: dellemc/csi-metadata-retriever:v1.8.0
+        image: quay.io/dell/container-storage-modulescsi-metadata-retriever:v1.8.0
```

Comment:
The image path is incorrect. Please correct the same.

Reply:
@harishp8889 Please address the above issue and verify other links as well.

Reply:
Update in the following PR: https://github.com/dell/csm-operator/pull/759, Please review